adsf/bin/adsf

```diff
@@ -29,6 +32,10 @@ OptionParser.new do |opts|
     options[:index_filenames] = o.split(',')
   end
 
+  opts.on('-x', '--auto-extensions [extensions]', 'Suffixes to automatically apply to requests (comma-separated, including dot)') do |o|
```

Comment:
```suggestion
  opts.on('-x', '--auto-extensions [extensions]', 'Specify suffixes to automatically apply to requests (comma-separated, including dot)') do |o|
```

This makes the option descriptions consistent (they all use the imperative mood).